index.js

```diff
@@ -173,3 +173,26 @@ form.addEventListener('submit', (e) => {
     e.run();
   }
 });
+
+const content = () => {
```

Comment:
[OPTIONAL] It is a good practice to name your function in the [imperative](https://www.30secondsofcode.org/articles/s/javascript-naming-conventions) for. For example `saveInputs` instead of `content`.